front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/AboutReportWidget/index.tsx

```diff
@@ -41,9 +65,17 @@ const AboutReportWidget: UserComponent = ({
     ? null
     : `${user.attributes.first_name} ${user.attributes.last_name}`;
 
-  // TODO: initialise these from the report projects when available
-  const project = 'Project name';
-  const projectPeriod = '12/03/2022 - 15/12/2022';
+  // Project name & time period
+  const project = useProject({ projectId });
+  const phases = usePhases(projectId);
+  const projectName = isNilOrError(project)
+    ? ''
+    : localize(project.attributes.title_multiloc);
+  let projectPeriod = formatMessage(messages.continuousProject);
+  if (!isNilOrError(phases) && phases.length !== 0) {
+    const { startDate, endDate } = getPhaseDates(phases);
+    projectPeriod = `${startDate} - ${endDate}`;
+  }
```

Comment:
Small preference, but I would avoid using `let` as much as possible, especially in react components. E.g.

```ts
// outside of the component
const toPeriodString = ({ startDate, endDate }: { startDate: string; endDate: string; }) => `${startDate} - ${endDate}`; 

// in the component from line 75
const hasPhases = !isNilOrError(phases) && phases.length !== 0;
const projectPeriod = hasPhases
  ? toPeriodString(getPhaseDates(phases))
  : formatMessage(messages.continuousProject);
```